css/style.css

```diff
@@ -1,7 +1,242 @@
-body{
-    margin: 0 1rem;
-}
+
+/* Extra small devices (phones, less than 768px) */
 .header{
-    background-color: #F72210;
-    color: #FFF;
-}
\ No newline at end of file
+  justify-content: start;
+}
+.main-header {
+  font-size: 1.3rem;
+}
+.navbar-toggler-icon{
+  background-image: url("../images/hamburger_button.png");
+}
+.header-text a{
+  font-size: .7rem;
+}
+.bg-color01{
+  background-color: #C2D4D7;
+}
+.icon-search{
+  width: 2rem;
+  height: auto;
+}
+.logo{
+  width: 2rem;
+  height: 2rem;
+}
+.icon-slideshow{
+  width: 4rem;
+  height: 3rem;
+  background-image: url(https://g.newsweek.com/themes/newsweek/images/icon-slideshows-nw.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  text-indent: .4rem;
+  line-height: 2.5rem;
+}
+.cover-back, .cover-front{
+  width: 170px;
+  height: 228px;
+}
+.cover-front{
+  left: 4rem;
+  top: 2rem;
+}
+
+.subscribe, .newsletter-part{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+.subscribe-img{
+  flex: none;
+  padding-top: 1rem;
+  padding-bottom: 2rem;
+  margin-left: -3.5rem;
+}
+.newsletter-part img{
+  width: 100%;
+  height: 10rem;
+  object-fit: cover;
+}
+.copyright-social{
+  width: 95%;
+  height: auto;
+}
+.footer-social, .info-list, .editions-list{
+  list-style: none;
+  margin-bottom: .3rem;
+}
+.editions-list{
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+.footer-social li {
+  display: inline;
+}
+
+.info-list li, .editions-list li{
+  display: inline;
+  padding-right: .5rem;
+  margin-right: .5rem;
+}
+.info-list li:last-child, .editions-list li:last-child{
+  border-right: none
+}
+.info-list li a, .editions-list li a{
+  color: #6C757D;
+  font-size: .8rem;
+}
+
+.social-icon{
+  width: 2rem;
+  height: auto;
+  filter: grayscale(100%);
+}
+.social-icon:hover{
+  filter: grayscale(0%);
+}
+.copyright{
+  font-size: .8rem;
+}
+.footer-editions{
+  justify-content: start;
+}
+
+/* Small devices (desktops, 576px and up) */
+@media (min-width: 576px){
+  .sponsor-part, .special-edition{
+      width: 50%;
+  }
+  .sign-up{
+    width: 45%;
+    height: auto;
+  }
+  .header-text a{
+    font-size: 1rem;
+  }
+}
+
+/* Medium devices (desktops, 768px and up) */
+@media (min-width: 768px) {
+  .main-header{
+    font-size: 2rem;
+  }
+  .subscribe{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    text-align: start;
+    padding-top: 0; 
+    margin-left: 0;
+  }
+  .newsletter-part{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

Reply:
done